data/array/filter/a3f6c365-be00-4e30-9772-877f6c38c4f3.json

```diff
@@ -1,27 +1,27 @@
 {
-  "description": "Filters array. External node(s) receive each item and decide whether to include them in the result.",
+  "description": "Filters array. External node(s) receive each item and decide whether to include them in the result.\n\nExample:\nhttps://github.com/Cranq-io/cranq-tutorials/blob/main/reference/3_querying_data/3_2_filters/README.md#using-filters",
```

Comment:
"Filters **the** array" or "Filters **an** array" or "Filters `array`" ?
For me something is missing here.